data/core/stories.md

```diff
@@ -51,6 +57,16 @@
     - utter_great
     - utter_anything_else
 
+## newsletter + ask why email
+* greet
+    - action_greet_user
+* signup_newsletter
+    - utter_great
+    - utter_ask_email
+* explain
```

Comment:
i think this is a great addition, but maybe we should wait for Ellas PR to be merged because once we restructure to forms things will be different :D